experimental/builder/test/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(CMAKE_CXX_STANDARD 20)
+
+include(gtest)
+
+# Helper function to create a gtest executable with common properties
+function(add_ck_builder_test test_name)
+    add_executable(${test_name} ${ARGN})
+    target_include_directories(${test_name} PRIVATE
+        "${PROJECT_SOURCE_DIR}/experimental/builder/include"
```

Comment:
This directory doesn't exist, causing the build to fail. I suggest to create it with a `.gitkeep` file or something.